code/01-exploration.R

```diff
@@ -0,0 +1,96 @@
+library(here)
+library(tidyverse)
+
+# Read in data
+sugary_bev <- read.csv(file = "./rawdata/june1data.csv", 
+  colClasses = c("DofW" = "factor", "Site" = "factor", "Intervention" = "factor"))
+
```

Comment:
there is no exploration of missing data. As a starter, you can use visdat package to visualize the missingness.